From 1fc106e93554f848fac737a69eb539737cca96d7 Mon Sep 17 00:00:00 2001
From: Ren Jianing <jianing.ren@rock-chips.com>
Date: Fri, 20 Mar 2020 10:16:23 +0800
Subject: [PATCH] ARM: dts: rockchip: enable usb for rv11xx-evb-v10

This patch makes OTG port work in peripheral mode only. In addition,
note that Host port and the Naneng USB 2.0 driver is not enabled.

Change-Id: I2dad596115688d08cb091596a1bb9b905612b3c6
Signed-off-by: Ren Jianing <jianing.ren@rock-chips.com>
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 86bfd6627b5f..b35a35f4ac5c 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -437,3 +437,13 @@
 	supports-sdio;
 	status = "okay";
 };
+
+&usbdrd {
+	status = "okay";
+
+	usbdrd_dwc3 {
+		status = "okay";
+		/delete-property/ phys;
+		dr_mode = "peripheral";
+	};
+};
-- 
2.35.3

